README.rst

```diff
@@ -127,7 +101,7 @@ See the guidelines for contributing in ``CONTRIBUTING.rst``.
 Acknowledgments
 ---------------
 
-The PyGSP was started in 2014 as an academic open-source project for
+The PyGSP2 was started in 2014 as an academic open-source project for
```

Comment:
Reference to the original project. We should not change this.

Reply:
solved